core/commons/che-core-commons-observability/src/main/java/org/eclipse/che/commons/observability/CountedThreadFactory.java

```diff
@@ -45,19 +45,19 @@ public CountedThreadFactory(
     this.created =
         Counter.builder("thread.factory.created")
             .tags(Tags.concat(tags, "name", name))
-            .description("The approximate number of threads that create with thread factory")
+            .description("The approximate number of threads which were created with a thread factory")
```

Comment:
```suggestion
            .description("The approximate number of threads that were created with a thread factory")
```
(It's a defining clause.)

Reply:
In fact, we can leave it out completely:
```suggestion
            .description("The approximate number of threads created with a thread factory")
```